lib/init/config-initializer.js

```diff
@@ -180,7 +180,7 @@ function processAnswers(answers) {
         config.extends.push("plugin:react/recommended");
     } else if (answers.framework === "vue") {
         config.plugins = ["vue"];
-        config.extends.push("plugin:vue/vue3-essential");
+        config.extends.push("plugin:vue/vue3-recommended");
```

Comment:
we intentionally only enable the very basic rules, as `--init` is mostly the first step for the new users. but I'm fine with the change if the vue team think `"vue3-recommended"` should be applied for most vue users.

@ota-meshi, would very like to see your thoughts. :)

Reply:
Personally, I recommend `vue3-recommended` for users who are somewhat familiar with eslint itself. But I think `vue3-essential` is a better first step. So I think it makes sense for `eslint/create-config` to set `vue3-essential`.

Reply:
Hi @aladdin-add and @ota-meshi, thank you for your response!

That sounds like a reasonable option! I will close this merge request. I will also be raising another proposal for this project shortly, so please take a look when you have time.